charts/akeyless-api-gateway/templates/tls.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if ( and .Values.cache .Values.cache.tls .Values.cache.tls.autoGenerated) }}
+{{- $fullname := .Release.Name | trunc 63 | trimSuffix "-" -}}
+{{- $ca := genCA "redis-ca" 365 }}
+{{- $altNames := list $fullname "127.0.0.1" "localhost" (include "cache_address" .) (printf "*.%s" (include "cache_address" . ))}}
+{{- $cert := genSignedCert $fullname nil $altNames 365 $ca }}
+{{- $existingSecret := (lookup "v1" "Secret" .Release.Namespace (include "generated_cache_tls_secret_name" .)).data }}
```

Comment:
i suggest to to remove lookup or to mention in the readme / docs that if client using gitops tools, they can only used with existing secret. please sync with product @renanaAkeyless

Reply:
Whether the lookup is there or not it will still reconcile and recreate the files, so I recommend we document it
@EldadH89 how do you want to formulate it in the readme?